src/components/ui/button/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { attachClasses } from '../../../utils/index';
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine